### DISCUSSION TITLE
是否支持表配置？

### DISCUSSION THREAD

**Discussion by edisenwangK** [apolloconfig/apollo#3888] [Q&A]
- [x] 我已经检查过[discussions](https://github.com/ctripcorp/apollo/discussions)
- [x] 我已经搜索过[issues](https://github.com/ctripcorp/apollo/issues)
- [x] 我已经仔细检查过[FAQ](https://www.apolloconfig.com/#/zh/faq/common-issues-in-deployment-and-development-phase)

我在使用demo环境时，发现配置仅支持KEY-VALUE的方式进行配置，是否有支持TABLE方式的配置？

例如类似MYSQL里，有多行（记录）数据的场景

**Comment by nobodyiam** [comment]
目前支持的是基本的 k-v 配置，可以尝试基于这个模型在上层封装 table 模型的配置，比如增加一种 namespace 的类型
